### ISSUE TITLE
"Table does not exist" error when query is executed after "Key already exists" error

### ISSUE THREAD

**Issue by mvallet91** [ujjwalguptaofficial/sqljs#2]
I'm using this great library together with JsStore, in an iterative process of reading log records from many JSON files, processing and then storing into IndexedDB via SqlWeb + JsStore, and it works great. 

However, like the title says, if I get an error of "key already exists", when the script moves to the next query (often on a different table), I get the error of "table does not exist", when it definitely exists.

I use **connection.runSql(query).then(...).catch(...)** but it still raises the table error for all following queries, until the whole process eventually stops.

![image](https://user-images.githubusercontent.com/15776309/55954606-6bb23380-5c5f-11e9-9830-ba1913c3f038.png)

**Comment by ujjwalguptaofficial** [comment]
Seems interesting. I will try to reproduce this & if you have something that you can give me, that would be great.

**Comment by mvallet91** [comment]
A brief explanation of what my script does: read logfile with records in JSON format, put the records into a list, process the list to find a type of events, say *sessions*, and insert to a *sessions* table via SqlWeb like this: 
```
let query = new SqlWeb.Query("INSERT INTO " + table + " values='@val'");
query.map("@val", data);
connection.runSql(query)
   .then(function (rowsAdded) { if (rowsAdded > 0) { console.log('Success'); } })
   .catch(function (err) { console.log(err);  });
```
And then do the same for other types of events in the same list of records, then repeat for many JSON, everything on client side.

Sometimes if I get the "key already exists" error, it is caught and it continues working, but other times I get the "table not exists error" for all the following tables until it completely crashes, or this one, which instantly stops the whole process:

![image](https://user-images.githubusercontent.com/15776309/56023229-bc806580-5d0d-11e9-830d-28fc923626c0.png)

Let me know if there's anything I can tell you!

**Comment by ujjwalguptaofficial** [comment]
There was a small query flow issue with jsstore. I have fixed it and released it in new version - 2.11.0. Please install the latest version of jsstore and let me know if fixed or still getting the issue.

**Comment by mvallet91** [comment]
Thank you! I've been trying to reproduce the error and it seems to be fixed. 
Will let you know if it comes up again.
